sample_file.py

```
@@ -0,0 +1,47 @@
+
+def infinite_loop():
+    while True:
+        print("This is an infinite loop")
+
+def sum_numbers(numbers):
+    total = 0
+    for num in numbers:
+        total += num
+    return total
+
+def divide(a, b):
```

Ruff

Missing return type annotation for public function `divide`

(ANN201)